src/cpp/rtps/writer/StatelessWriter.cpp

```diff
@@ -166,12 +175,12 @@ bool StatelessWriter::change_removed_by_history(CacheChange_t* change)
     return true;
 }
 
-bool StatelessWriter::is_acked_by_all(const CacheChange_t* change) const
+bool StatelessWriter::is_acked_by_all(const CacheChange_t* change)
```

Comment:
If `const` was removed because the mutex, make it mutable instead of removing the `const`.